src/coreclr/tools/Common/JitInterface/CorInfoImpl.cs

```diff
@@ -366,6 +366,18 @@ private void CompileMethodInternal(IMethodNode methodCodeNodeNeedingCode, Method
 #endif
             }
 
+            if (codeSize != _code.Length)
```

Comment:
should this check be `<` instead of `!=`?

Reply:
I had that in the assert, but probably sensible to have `<` instead. I will update it.